backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
I don't think it makes much difference either way as to method name/separate vs same method -- `admin_layout=` is still a method of course. If it's `admin_layout=`, it doesn't really matter if you do it with `attr_writer` or not, it's irrelevant, whatever works best. 

I don't personally have a problem with the same method being a setter and getter, in this context, but I see the argument that a separate `=` method might be more clear. I think either are fine. 

I agree with @jhawthorn please don't use instance variables for API/contract between two entities, even though Rails sometimes seems to encourage it. As long as you don't do that.